<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>log_in</class>
 <widget class="QWidget" name="log_in">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>361</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>50</number>
    </property>
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="log_in_confirm">
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="user_name">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777212</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 用户名</string>
      </property>
      <property name="margin">
       <number>-2</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="user_name_input"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>361</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="password">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777212</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 密   码 </string>
      </property>
      <property name="margin">
       <number>-2</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password_input"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="error_sign">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>180</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>8</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>用户名或密码错误，请检查您的输入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
